internal/adapters/uidriver/socket.go

```
@@ -0,0 +1,234 @@
+
+// Bridge connects stdio to an opted-in interactive UI socket without creating
+// another attachment.
+func Bridge(ctx context.Context, path string, input io.Reader, output io.Writer) error {
+	if err := validateSocketPath(path); err != nil {
+		return err
+	}
+	conn, err := (&net.Dialer{}).DialContext(ctx, "unix", path)
+	if err != nil {
+		return fmt.Errorf("uidriver: connect socket: %w", err)
+	}
+	defer conn.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)